tests/integration_tests/test_login.py

```diff
@@ -221,3 +224,109 @@ async def test_server_double_login(lobby_server):
         "style": "error",
         "text": "You have been signed out because you signed in elsewhere."
     }
+
+
+async def test_server_valid_login_with_token(lobby_server, jwk_priv_key):
+    proto = await connect_client(lobby_server)
+    await proto.send_message({
+        "command": "auth",
+        "version": "1.0.0-dev",
+        "user_agent": "faf-client",
+        "token": jwt.encode({
+          "sub": 3,
+          "user_name": "Rhiza",
+          "scope": [],
+          "exp": int(time() + 1000),
+          "authorities": [],
+          "non_locked": True,
+          "jti": "",
+          "client_id": ""
+        }, jwk_priv_key, algorithm="RS256", headers={"kid": "L7wdUtrDssMTb57A_TNAI79DQCdp0T2-KUrSUoDJBhk"}),
```

Comment:
Maybe it makes sense to define the jwk as a fixture as well, then we don't need to hardcode the kid here

Reply:
You mean the header for the jwk? As each test has a slightly different jwk.

Reply:
I mean the kid really. The giant string that's the same in every test.